### PR TITLE
Merge master changes to develop

### PR DESCRIPTION
**Pull request by pploegaert**

### CODE REVIEW

File: ci/scenario_helpers/fwk_handler.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+class FwkHandler(object):
+    """
+    Class handling fwk actions
+    """
+    LOGGER = LogHandler.get(source='scenario_helpers', name='fwk_handler')
+
+    @classmethod
+    def restart(cls, srs, logger=LOGGER):
+        for sr in srs:
```
Comment (JeffreyDevloo) [inline ci/scenario_helpers/fwk_handler.py]: make srs storagerouters

File: ci/scenario_helpers/fwk_handler.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            logger.info("Restarting ovs-workers on {0}".format(sr.ip))
+            client = SSHClient(str(sr.ip), username='root', cached=False)
```
Comment (JeffreyDevloo) [inline ci/scenario_helpers/fwk_handler.py]: No need to do the ip stuff here.
SSHClient(storagerouter, username=root)

Comment (pploegaert) [inline ci/scenario_helpers/fwk_handler.py]: I know but that did not work ... as that uses the heartbeat ... and caused an exception not being able to connect ...

File: ci/scenario_helpers/fwk_handler.py (hunk 1 above)
Comment (JeffreyDevloo) [inline ci/scenario_helpers/fwk_handler.py]: Or add in some docstrings

**Pull request merged**